packages/server/src/notification/notification.service.ts

```diff
@@ -88,16 +95,70 @@ export class NotificationService {
   }
 
   // notifies a user via phone number
-  async notifyPhone(pn: PhoneNotifModel, message: string): Promise<void> {
-    try {
-      this.twilioClient &&
-        (await this.twilioClient.messages.create({
-          body: message,
-          from: this.configService.get('TWILIOPHONENUMBER'),
-          to: pn.phoneNumber,
-        }));
-    } catch (error) {
-      console.error('problem sending message', error);
+  async notifyPhone(
+    pn: PhoneNotifModel,
+    message: string,
+    verificationMethod: boolean,
+  ): Promise<void> {
+    if (verificationMethod || pn.verified) {
+      try {
+        this.twilioClient &&
+          (await this.twilioClient.messages.create({
+            body: message,
+            from: this.configService.get('TWILIOPHONENUMBER'),
+            to: pn.phoneNumber,
+          }));
+      } catch (error) {
+        console.error('problem sending message', error);
+      }
+    }
+  }
+
+  async verifyPhone(
+    phoneNumber: string,
+    message: string,
+    response: Response,
+  ): Promise<void> {
+    const MessagingResponse = require('twilio').twiml.MessagingResponse;
+    const twiml = new MessagingResponse();
+
+    if (message !== 'YES' && message !== 'NO' && message !== 'STOP') {
+      twiml.message(
+        'Please respond with either YES or NO. Text STOP at any time to stop receiving text messages',
```

Comment:
Can we put all these messages in a constants file so that we can edit the copy easier